content/2018-04-17_mame-python-v-hradci-kralove.md

```diff
@@ -48,6 +48,7 @@ Těší mě, že kurz měl úspěch nejen u účastníků, ale také u koučů,
 
 Držte nám palce!
 
+Máte-li zájem získávat více informací o pythoním dění v Hradci Králové můžete nás sledovat na [stránkách Facebooku](https://www.facebook.com/groups/pythonhradeckralove/).
```

Comment:
```suggestion
Máte-li zájem získávat více informací o pythoním dění v Hradci Králové můžete nás sledovat na [Facebookové skupině](https://www.facebook.com/groups/pythonhradeckralove/).
```

Možná bych to napsal spíš takto, ale schvaluji i v původním znění.

Reply:
Podle mě by mělo být "ve skupině" spíš než "na skupině" :)